src/BusinessLayer/DataPopulators/AuxBmsPopulator/AuxBmsPopulator.cpp

```diff
@@ -20,6 +20,21 @@ void AuxBmsPopulator::populateData(const QJsonObject& data)
     auxBmsData_.setAuxBmsAlive(value.toObject().value(JsonFormat::AUXBMS_AUXBMSALIVE).toBool());
     auxBmsData_.setStrobeBmsLight(value.toObject().value(JsonFormat::AUXBMS_STROBEBMSLIGHT).toBool());
     auxBmsData_.setAllowCharge(value.toObject().value(JsonFormat::AUXBMS_ALLOWCHARGE).toBool());
-    auxBmsData_.setContactorError(value.toObject().value(JsonFormat::AUXBMS_CONTACTORERROR).toBool());
-    auxBmsData_.setHighVoltageEnable(value.toObject().value(JsonFormat::AUXBMS_HIGHVOLTAGEENABLE).toBool());
+    auxBmsData_.setChargeTripHighVolt(value.toObject().value(JsonFormat::AUXBMS_CHARGETRIP_HIGHVOLT).toBool());
+    auxBmsData_.setChargeTripPackCurr(value.toObject().value(JsonFormat::AUXBMS_CHARGETRIP_PACKCURR).toBool());
+    auxBmsData_.setChargeTripHighTempCurr(value.toObject().value(JsonFormat::AUXBMS_CHARGETRIP_HIGHTEMPCURR).toBool());
+    auxBmsData_.setDischargeTripLowVolt(value.toObject().value(JsonFormat::AUXBMS_DISCHARGETRIP_LOWVOLT).toBool());
+    auxBmsData_.setDischargeTripPackCurr(value.toObject().value(JsonFormat::AUXBMS_DISCHARGETRIP_PACKCURR).toBool());
+    auxBmsData_.setDischargeTripHighTempCurr(value.toObject().value(JsonFormat::AUXBMS_DISCHARGETRIP_HIGHTEMPCURR).toBool());
+    auxBmsData_.setProtectionTrip(value.toObject().value(JsonFormat::AUXBMS_PROTECTIONTRIP).toBool());
+    auxBmsData_.setHighVoltageEnableState(value.toObject().value(JsonFormat::AUXBMS_HIGHVOLTAGE_ENABLESTATE).toBool());
+    auxBmsData_.setAllowDischarge(value.toObject().value(JsonFormat::AUXBMS_ALLOW_DISCHARGE).toBool());
+    auxBmsData_.setOrionCANReceivedRecently(value.toObject().value(JsonFormat::AUXBMS_ORION_CAN_RECEIVED_RECENTLY).toBool());
+    auxBmsData_.setContactorErrorCharge(value.toObject().value(JsonFormat::AUXBMS_CHARGE_CONTACTORERROR).toBool());
+    auxBmsData_.setContactorErrorDischarge(value.toObject().value(JsonFormat::AUXBMS_DISCHARGE_CONTACTORERROR).toBool());
+    auxBmsData_.setContactorErrorCommon(value.toObject().value(JsonFormat::AUXBMS_COMMON_CONTACTORERROR).toBool());
+    auxBmsData_.setDischargeShouldTrip(value.toObject().value(JsonFormat::AUXBMS_DISCHARGE_SHOULD_TRIP).toBool());
+    auxBmsData_.setChargeShouldTrip(value.toObject().value(JsonFormat::AUXBMS_CHARGE_SHOULD_TRIP).toBool());
+    auxBmsData_.setChargeOpenButShouldBeClosed(value.toObject().value(JsonFormat::AUXBMS_CHARGE_OPEN_BUT_SHOULD_BE_CLOSED).toBool());
+    auxBmsData_.setDischargeOpenButShouldBeClosed(value.toObject().value(JsonFormat::AUXBMS_DISCHARGE_OPEN_BUT_SHOULD_BE_CLOSED).toBool());
```

Comment:
> Also, this was kind of an issue beforehand, but it's more pronounced now - the battery stuff's alignment is pretty bad. I think we should try and line them up better this PR.

I think that is out of scope for this story. I believe it would be a better idea to make a new story to clean up the ui, that way we can clean up a bunch of other stuff at the same time.

Reply:
- Reordered the tabs
- I prefer Alive label outside.
- I created a story for fixing battery page alignment
https://jira.calgarysolarcar.ca/browse/SFT-148